spec/oppiabotSpec.js

```diff
@@ -102,24 +119,18 @@ describe('Oppiabot\'s', () => {
       expect(apiForSheetsModule.checkClaStatus.calls.count()).toEqual(1);
     });
 
-    it('should be called with three arguments for the given payload', () => {
+    it('should be called with one argument for the given payload', () => {
```

Comment:
Can you please explain this change?

Reply:
`apiForSheetsModule.checkClaStatus` expects to be called with just one argument `Context`
https://github.com/oppia/oppiabot/blob/04d7f3b8723165bbbdc51074855d5223af75b2cf/index.js#L27

Reply:
Ok, sg.